src/templates/ratings/add.html

```diff
@@ -7,6 +7,11 @@
           {% endif %}>
       {% if isModal %}
         <h3>{{ existingReview and _('Edit Review') or _('Leave a Review') }}</h3>
+	<p class="reviews--guideline">
+          {{ _('Please read the <a href="{url}">Review Guidelines</a> for more details about rating apps.
```

Comment:
Just to open in new tab, target="_blank" is better, right? But I read rel="external" is still better thought it will not open link in new tab by default. Little confused here about what to do.
ref - http://www.copterlabs.com/blog/easily-create-external-links-without-the-target-attribute/